source/ht_file.cpp

```diff
@@ -183,15 +162,12 @@ namespace Hatchit
         **/
         bool File::Close(void)
         {
-            int ret = 0;
-            //Close file if handle is active
-            if (m_handle)
+            if (m_handle.is_open())
             {
-                ret = fclose(m_handle);
-                m_handle = nullptr;
+                m_handle.close();
+                return true;
             }
-
-            return (ret <  0) ? false : true;
+            return false;
```

Comment:
Just always return true if the function is noexcept.  If the function isn't noexcept, then ... either try-catch and return false on catch, or pass the error upward?  Not 100% sure which is best.